### PR TITLE
Custom conditions

### PR DESCRIPTION
**Pull request by NikitaKhmelevsky**

### CODE REVIEW

**Comment by astyagun** [comment]
Why is this PR in this repository?

**Comment by NikitaKhmelevsky** [comment]
@astyagun do we have other repository for this task?

**Comment by astyagun** [comment]
No, but one should've been requested and work should've been pushed there. I'll create one now.

**Comment by astyagun** [comment]
https://github.com/jetruby/rails-production_console_warning

File: README.md [hunk 1]
```diff
@@ -3,21 +3,26 @@
...
 ## Installation
 
 Add this line to your application's Gemfile:
 
 ```ruby
 gem 'rails-production_console_warning'
+
+or
+
+gem 'rails-production_console_warning', path: 'your/local/path'
```
Comment (astyagun) [inline README.md:48]: No need in this. I think, this documentation is for users of the gem, not for it's developers.

**Comment by astyagun** [comment]
Please resubmit this PR into JetRuby repository

**Pull request merged**